apps/doop/src/components/filters/FilterSelect.tsx

```diff
@@ -57,22 +64,17 @@ const FilterSelect = () => {
             }
           </Select>
           <ComboBox
+            key={filterValue}
```

Comment:
[suggestion]: to add comment here so people know that this is temporary workaround until we fix combox UX.